content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Quickstart: SCITT for Supply Chains (Preview)"
+description: "Getting Started with SCITT, enabling Software Supply Chain scenarios (Preview)"
+lead: "Push a collection of Supply Chain Statements using SCITT APIs"
+date: 2021-06-09T13:49:35+01:00
+lastmod: 2021-06-09T13:49:35+01:00
+draft: false
+images: []
+menu:
+  developers:
+    parent: "developer-patterns"
+weight: 112
+toc: true
+aliases: 
+  - /docs/developer-patterns/scitt-api/
+---
+
+{{< caution >}}
+The SCITT API is currently in preview and subject to change
+{{< /caution >}}
+
+The **S**upply **C**hain **I**ntegrity, **T**ransparency and **T**rust (SCITT) initiative is a set of [IETF standards](https://datatracker.ietf.org/group/scitt/documents/) for managing the compliance and auditability of goods and services across end-to-end supply chains.
+SCITT supports the ongoing verification of goods and services where the authenticity of entities, evidence, policy, and artifacts can be assured and the actions of entities can be guaranteed to be authorized, non-repudiable, immutable, and auditable.
+
+To assure insights to supply chain artifacts are current, the SCITT APIs provide a correlation of statements for a specific artifact, allowing newer information to be registered for the most up to date insights.
+
+This quickstart will:
+
+1. create, or use an existing a key to sign a collection of statements about an artifact
+1. create and register an SBOM for the artifact
+1. create and register an attestation for the artifact
+1. query a collection of statements about the artifact
+
+## Prerequisites
+
+- [A DataTrails subscription](https://app.datatrails.ai/signup)
+- [DataTrails sample code](#datatrails-sample-code)
+- [SBOM Tool](https://github.com/microsoft/sbom-tool)
+
+### DataTrails Sample Code
+
+Clone the [DataTrails SCITT Examples](https://github.com/datatrails/datatrails-scitt-samples) repository
+
+```bash
+git clone https://github.com/datatrails/datatrails-scitt-samples.git
+
+cd datatrails-scitt-samples
+```
+
+## Create a Signing Key
+
+{{< note >}}
+If you already have a COSE Key, skip ahead to [Register a SBOM for the Artifact](#register-a-sbom-for-the-artifact)
+{{< /note >}}
+
+DataTrails supports multiple signing keys.
+_(TODO: link to supported DataTrails Signing Keys\)_<br>
+For a quickstart, we'll create a testing [COSE Key](https://cose-wg.github.io/cose-spec/#key-structure) which DataTrails will cryptographically validate upon registration
+
+1. Create a local signing key
+
+    ```shell
+    openssl ecparam -name prime256v1 -genkey -out scitt-signing-key.pem
+    ```
+
+## Register a SBOM for the Artifact
+
+When registering statements about an artifact, a common identifier is required to correlate a collection of statements.
+In SCITT, a `CWT_Claims subject` property captures the identifier.
```

Comment:
Minor: I am still adamant that at the moment it is feed, we need to document what the implementation currently does.

We need to ensure we are accurate, I am happy to say both, but we need to say somewhere that the correlation of statements is currently implemented under the `feed` header.

Reply:
Yes, it is understood that what was a special tag called feed in the protected header is in future moving to CWT claims sub field. But we do need a quickstart to acurately reflect the current product. A note acknowleging our commitment to aligning with the standard would ideally be quite prominent tho:

```suggestion

When registering statements about an artifact, a common identifier is required to correlate a collection of statements. To accomplish this today, we use the feed property of the protected header

**_NOTE:_**  SCITT is an evolving standard and in the lastest work, a `CWT_Claims subject` property captures the identifier. And this platform will be aligning in a future release
```